packages/alfa-dom/src/get-tab-sequence.ts

```diff
@@ -0,0 +1,46 @@
+import { getTabIndex } from "./get-tab-index";
+import { traverseNode } from "./traverse-node";
+import { Element, NodeType } from "./types";
+
+function binaryIndexSearch(array: Array<Element>, index: number) {
+  let low = 0;
+  let high = array.length;
```

Comment:
I tend to call these `lower` and `upper` for the simple reason that they align 😄